### PR TITLE
tss lookup plugin - Fetch secret by path

### PR DESCRIPTION
**Pull request by delinea-sagar**
Added support to fetch secret from secret server by path

### CODE REVIEW

File: changelogs/fragments/get-secret-by-path.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - tss lookup plugin - allow to fetch secret by path. Previously, we could not fetch secret by path but now use ``secret_path`` option to indicate to fetch secret by secret path.
```
Comment (felixfontein) [inline changelogs/fragments/get-secret-by-path.yml:2]:
```suggestion
  - tss lookup plugin - allow to fetch secret by path. Previously, we could not fetch secret by path but now use ``secret_path`` option to indicate to fetch secret by secret path (https://github.com/ansible-collections/community.general/pull/6881).
```

File: plugins/lookup/tss.py [hunk 2]
```diff
@@ -26,6 +26,11 @@
         description: The integer ID of the secret.
         required: true
         type: int
+    secret_path:
+        description: Indicate a full path of secret including folder and secret name when the secret ID is set to 0.
```
Comment (felixfontein) [inline plugins/lookup/tss.py:30]: The UX of this feels strange. Using a magic value `0` to specify the real value somewhere else seems very confusing.

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

File: plugins/lookup/tss.py (hunk 2 above)
Comment (felixfontein) [reply plugins/lookup/tss.py]: You removed that part from the description, but in the example and the code the magic value is still there.

Comment (delinea-sagar) [reply plugins/lookup/tss.py]: Actually, Get secret API of Delinea secret server required input in this format when fetching secret by path.

**Review by delineaKrehl — APPROVED** [review]

File: plugins/lookup/tss.py (hunk 2 above)
Comment (felixfontein) [reply plugins/lookup/tss.py]: Then why does the documentation not say this? (But only the example and the code itself?)

Comment (delinea-sagar) [reply plugins/lookup/tss.py:30]: @felixfontein added in documentation also

**Comment by felixfontein** [comment]
@delinea-sagar thanks for your contribution!
@delineaKrehl thanks for reviewing!

**Pull request merged**